src/libopensc/opensc.h

```diff
@@ -1731,6 +1746,20 @@ iso7816_build_pin_apdu(struct sc_card *card, struct sc_apdu *apdu,
  */
 void sc_free(void *p);
 
+#ifdef ENABLE_OPENSSL
+#include <openssl/evp.h>
```

Comment:
I would vote for having the includes on the top of the file, instead of having them buried in here, but if others are ok with this, I can live with that.